CHT/SinkTemperature.C

```diff
@@ -32,15 +32,15 @@ void preciceAdapter::CHT::SinkTemperature::write(double* buffer, bool meshConnec
         tmp<scalarField> patchInternalFieldTmp = TPatch.patchInternalField();
         const scalarField& patchInternalField = patchInternalFieldTmp();
 
-        //If we use the mesh connectivity, we interpolate from the centres to the nodes
+        // If we use the mesh connectivity, we interpolate from the centres to the nodes
```

Comment:
Let's merge `develop` to the `volume_coupling` branch first, then the `volume_coupling` branch to this branch, and then check the formatting again. Maybe some of these changes have already been introduced before.